src/_gettsim/parameters/unterhaltsvors.yaml

```diff
@@ -45,7 +45,7 @@ altersgrenzen:
       payments. In addition, children up to the age of 18 receive advance on alimony
       payments if the parent has income beyond a minimal threshold.
   reference: § 1 Abs. 1, 1a UhVorschG
-  2017-01-01:
+  2009-01-01:
```

Comment:
In July 2017 there passed a law that introduced the third age group (12 to 18): [Artikel 23 G. v. 14.08.2017 BGBl. I S. 3122](https://www.buzer.de/gesetz/12769/a209095.htm)

The function with all three age groups in the unterhaltsvorschuss module starts in January 2017. Can you maybe look up what happened between January and July 2017?

Reply:
Thanks. I updated the law references and the start/end dates. Also tested the synthetic households again for 2016-01, 2017-01, and 2017-07 (updated tables in PR description).